tests/unit-tests/admin/test-class-sensei-setup-wizard.php

```diff
@@ -269,6 +287,17 @@ public function testActivationRedirectWithoutTransient() {
 		$setup_wizard_mock->activation_redirect();
 	}
 
+	/**
+	 * Testing if redirect option is cleared on setup wizard rendering.
+	 */
+	public function testRenderWizardPageClearsRedirectOption() {
```

Comment:
Should we name the test function like we do for our other tests? Like this p6rkRX-35r-p2 (test**_When**_Does**)?

Reply:
Sure! I didn't do that to follow the old standard of this file, but to make it write I changed the whole file here now: https://github.com/Automattic/sensei/pull/7581/commits/2d7b8eb903dbb8dc00b466e502505cda51ce9c79